src/main/org/nlogo/window/ButtonWidget.scala

```diff
@@ -374,15 +373,22 @@ class ButtonWidget(random:MersenneTwisterFast) extends JobWidget(random)
     def paintButtonRectangle(g: Graphics) {
       g.setColor(getPaintColor)
       g.fillRect(0, 0, getWidth(), getHeight())
-      def renderImages(g: Graphics, dark: Boolean) {
+      def renderImages(g: Graphics, inverse: Boolean) {
         def maybePaintForeverImage() {
           if (forever) {
-            val image = if (dark) FOREVER_GRAPHIC_DARK else FOREVER_GRAPHIC
+            val image = new ImageIcon(java.awt.Toolkit.getDefaultToolkit.createImage(
```

Comment:
What exactly do you mean by "loss of flexibility between enabled/disabled modes" ?